test/unit/uk/gov/hmrc/bindingtariffclassification/search/SearchTest.scala

```diff
@@ -56,7 +56,7 @@ class SearchTest extends UnitSpec {
   "Search Binder" should {
 
     "Unbind Unpopulated Search to Query String" in {
-      Search.bindable.unbind("", Search()) shouldBe ""
+      Search.bindable.unbind("", Search()) shouldBe "sort_by=commodity-code&sort_direction=asc"
```

Comment:
This shouldn’t be the case, there’s no need for the backend to search by default